composer.json

```diff
@@ -26,7 +26,7 @@
   ],
   "require": {
     "php": "^7.1",
-    "illuminate/contracts": "^5.6",
+    "illuminate/contracts": "^5.6|^6.0",
```

Comment:
Hi, sorry for our late response and thanks for your contribution. I think we'd be be fine with

```
"illuminate/contracts": "^6.0"
```

Then we can create a new tag for this project.

If someone really needs to use an older version of `illuminate/contracts`, they can use the older version of this project :)

Would you mind doing that, plus squashing the change into a single commit? Thanks!

Reply:
sure I can fix that

Reply:
I think it's better to keep the 5.6 requirement as 6.* requires php 7.2

Reply:
see https://travis-ci.org/erasys/openapi-php/jobs/638492788?utm_medium=notification&utm_source=github_status

Reply:
I think we shouldn't support PHP 7.1 in any next versions, anyway. It is past "end of life" already. Only PHP 7.2+ is currently under maintenance.

Would it be a problem for you if this project would require `^7.2`?

<img width="1019" alt="Screenshot 2020-01-23 at 15 20 17" src="https://user-images.githubusercontent.com/6202537/72992339-dfd75580-3df3-11ea-9a22-9df628f7964a.png">

Reply:
ok, i'll remove the php 7.1 requirement from Travis. It's not a problem for me as otherwise I wouldn't need a 6.* requirement for illuminate/contracts :)